FireRender.Maya.Src/FireRenderVolume.cpp

```diff
@@ -441,19 +441,16 @@ bool NorthstarRPRVolume::TranslateVolume()
 			RPR_GRID_INDICES_TOPOLOGY_XYZ_U32
 		);
 
+		MFnDependencyNode depNode(node);
+		float maxDensityValue = RPRVolumeAttributes::GetDensityMultiplier(depNode) * 1.1f;
```

Comment:
Please make 1.1f multipler a constant and add a comment where 1.1 value comes from